inspire/dojson/common/base.py

```diff
@@ -39,6 +39,12 @@ def control_number(self, key, value):
     return value[0]
 
 
+@hep2marc.over('001', 'control_number')
+def control_number2marc(self, key, value):
+    """Record Identifier."""
+    return [value]
```

Comment:
Well you would need to teach `create_record` what are `controlfield`s and then you do additional test for all non-`controlfield`s. Moreover you loose flexibility if you would like to validate that any `controlfield` is not repeated. howgh